src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -698,7 +698,7 @@ namespace LowTempRadiantSystem {
                 }
             } else if (UtilityRoutines::SameString(Alphas(12), "CapacityPerFloorArea")) {
                 thisRadSys.CoolingCapMethod = CapacityPerFloorArea;
-                if (!lNumericBlanks(9)) {
+                if (!lNumericBlanks(11)) {
```

Comment:
Interesting that this wasn't causing any issues.  And interesting that no unit test caught this.  @RKStrand could you give a quick thumbs up/down on this?

@jmythms it would be really great to integrate testing of this in a unit test.  Is there an existing unit test that handles input processing of this that you could modify or mimic?  (Hint: there must be one because several lines of code around this line you changed are indeed covered by [unit test](http://energyplus.s3-website-us-east-1.amazonaws.com/coverage/2021-01-12-EnergyPlus-ec58f461aa-x86_64-gcc-7.5UTCD/EnergyPlus/LowTempRadiantSystem.cc.gcov.html).)

Reply:
Thank you for the review! Working on the unit test now. 

While working on this, I also just came across [this line](https://github.com/NREL/EnergyPlus/blob/ec58f461aa80708e4ef10c7d96e489bed71d5965/src/EnergyPlus/LowTempRadiantSystem.cc#L703), which I am not sure should have been 

`if (thisRadSys.ScaledCoolingCapacity <= 0.0) {` 
instead of 
`if (thisRadSys.CoolingCapMethod <= 0.0) {`.

I think this second change would generate the error message intended:
![image](https://user-images.githubusercontent.com/45446967/104359944-8b7a2980-54cd-11eb-8290-86b765d32630.png)

for this input object:
![image](https://user-images.githubusercontent.com/45446967/104360167-e3b12b80-54cd-11eb-995b-e833c46b3a28.png)

Reply:
@Myoldmopar I would agree that this is a bug that needs to be fixed.  It should be changed as @jmythms is suggesting.

My guess (and it's just a guess because I don't think I was the one to add all of the sizing mods that did things on a floor area, etc. basis) is that there simply isn't a test that exercises all of the potential options for both correct and incorrect input for low temperature radiant systems.  Is there one for a different input object like baseboards or something similar, maybe and if so, it would be good to implement such a unit testing scheme here.

Reply:
@jmythms Yes, I agree with your second recommended change also.  It should not be looking at CoolingCapMethod but ScaledCoolingCapacity.

Reply:
Thank you!